plugins/hypervisors/kvm/src/com/cloud/hypervisor/kvm/resource/LibvirtComputingResource.java

```diff
@@ -768,6 +770,12 @@ public boolean configure(final String name, final Map<String, Object> params) th
             _localStoragePath = "/var/lib/libvirt/images/";
         }
 
+        /* Directory to use for Qemu sockets like for the Qemu Guest Agent */
+        _qemuSocketsPath = (String)params.get("qemu.sockets.path");
+        if (Strings.isNullOrEmpty(_qemuSocketsPath)) {
```

Comment:
One issue we have found with ``Strings.isNullOrEmpty`` is that it does do a ``trim`` when checking for an empty string.  Therefore, a string containing one or more strings is considered non-empty.  Therefore, consider using ``StringUtils.isNoneBlank`` if a string containing only whitespace would be a a problem.